### DISCUSSION TITLE
Possibility to ignore() part of command code in dependency check?

### DISCUSSION THREAD

**Discussion by brendanf** [ropensci/targets#450] [Ideas]
Drake has `ignore()` as part of its DSL, to mark a part of a target command as unimportant for reproducibility.  I mostly used this for functions with internal parallelization, to pass the number of threads to use.  This value depends on the exact execution context, but does not change the result of the command.  I haven't been able to find any similar functionality in targets, so I am resorting to using environmental variables or `get()` in order to defeat introspection.  Is the lack of a `tar_ignore()` an intentional design decision? If so, what do you recommend for this specific use case?

**Comment by wlandau** [comment]
Your workaround seems fine for innocuous changes like the number of threads, etc. I thought about `ignore()` but decided against it for reasons I explain at https://github.com/ropensci/targets/issues/61#issuecomment-660636914.
